controllers/users/users.js

```diff
@@ -4,6 +4,43 @@ const jwt = require("jsonwebtoken");
 const config = require("config");
 const User = require("../../models/User");
 const messages = require("../messages");
+const nodemailer = require("nodemailer");
+require("dotenv").config();
+
+const sendVerificationEmail = user => {
+  const transporter = nodemailer.createTransport({
+    service: "gmail",
+    auth: {
+      user: process.env.GMAIL_USER,
```

Comment:
How developers should deliver this environment ? 
Should we send this credentials to all active developers to make allow them test this functionality ?

Reply:
Will be good idea to store content of .env in slack section?

Reply:
For now not so bad ;)